src/index.scss

```diff
@@ -0,0 +1,6 @@
+@import url('https://fonts.googleapis.com/css?family=Rajdhani:400,700&display=swap&subset=latin-ext');
+
+body {
+  font-family: 'Rajdhani', sans-serif;
+  font-weight: 400;
+}
```

Comment:
here you should have all scss imports 

// Components 
@import header.scss
@import weatherDetails.scs

etc